Source/WebCore/Modules/mediasource/SourceBufferList.cpp

```diff
@@ -67,9 +67,21 @@ void SourceBufferList::add(Ref<SourceBuffer>&& buffer)
     scheduleEvent(eventNames().addsourcebufferEvent);
 }
 
+bool SourceBufferList::contains(SourceBuffer& buffer) const
+{
+    return m_list.find(Ref { buffer } ) != notFound;
+}
+
+RefPtr<SourceBuffer> SourceBufferList::item(unsigned index) const
+{
+    if (index < m_list.size())
+        return m_list[index].copyRef();
+    return { };
+}
+
 void SourceBufferList::remove(SourceBuffer& buffer)
 {
-    size_t index = m_list.find(&buffer);
+    size_t index = m_list.find(Ref { buffer });
```

Comment:
find(buffer) is not working?

Reply:
unfortunately not. 
the definition and templating definition of find gave me an error